docs/logging-and-managing-experiment-results/logging-experiment-data.rst

```diff
@@ -0,0 +1,1587 @@
+.. _guides-logging-data-to-neptune:
+
+Logging experiment data
+=======================
+[loom-placeholder]
+
+During machine learning experimentation, you need to keep track of many different types of meta-data. Neptune helps you do that by logging, keeping track of and visualizing it.
+
+You can track many different types of data to the experiment. It can be metric, loss, image, interactive visualization, model checkpoint, pandas DataFrame and many more.
+
+Simply check :ref:`what you can log <what-you-can-log>` section below for a complete list.
+
+**On this page:**
+
+* :ref:`Basics of logging <basics-of-logging>`
+* :ref:`What you can log <what-you-can-log>`
+* :ref:`Advanced topics <logging-advanced>`
+
+.. _basics-of-logging:
+
+-----
+
+Basics of logging
+-----------------
+Logging experiments data to Neptune is simple and straightforward.
+
+Let's create minimal code snippet that logs single value to the experiment: ``'acc'=0.95``.
+
+.. code-block:: python3
+
+    import neptune
+
+    # Set project
+    neptune.init('my_workspace/my_project')
+
+    # Create experiment
+    neptune.create_experiment()
+
+    # Log 'acc' value 0.95
+    neptune.log_metric('acc', 0.95)
+
+Above snippet sets project, creates experiment and log one value to it. When script ends, the experiment is closed automatically. As a result you have new experiment with one value in one metric ('acc'=0.95).
+
+Everything that is evaluated after ``neptune.create_experiment()`` and before the end of the script (or when you call :meth:`~neptune.experiments.Experiment.stop`) can be logged to the experiment.
+
+.. _what-you-can-log:
+
+-----
+
+What objects can you log to Neptune
+-----------------------------------
+Neptune supports logging many different types of data. Here, you can find all of them listed and described.
+
+.. note::
+
+    Remember to set project using :meth:`~neptune.init` and :meth:`~neptune.projects.Project.create_experiment` before you start logging.
```

Comment:
This table doesn't look good. 
There is a lot of free room on the right -> and because if it is so long. 

Could you please rethink how to present it?

Reply:
made two column layout.